pymodels/BO_V06_01/__init__.py

```diff
@@ -17,4 +16,4 @@
 from .lattice import harmonic_number
 from .lattice import default_optics_mode, lattice_symmetry
 
-lattice_version = accelerator_data['lattice_version']
+lattice_version = 'BO_V06_01'
```

Comment:
Why are we defining this constant in two places? I think we could define it only once and use its value, don't we?

Besides, wouldn't it be nice to have the version of the git repository plus the last commit appended to this string?

Reply:
I think it is better indeed to define the constant in only one place. But for me the correct place seems to be in the subpackage's __init__.py module. It seems natural to me something like

```Python3
from pymodels import SI_V25_04 as si

print(si.lattice_version)
```

and not 

```Python3

from pymodels import SI_V25_04 as si

print(si.lattice.lattice_version)
```

since everything with the subpackage `si`, and not only accelerator data, is particular to a particular model version (which defines the subpackage). 

I think murilo and I discussed leaving the constant also in `accelerator_data`in order not to break existing code. But we may fix them, as they show up.

Reply:
Thanks for your comments and suggestions! Based on them, I made some changes in the code and updated the PR description, please take a look.